src/data/finset/basic.lean

```diff
@@ -2315,6 +2315,9 @@ protected def bUnion (s : finset α) (t : α → finset β) : finset β :=
 @[simp] theorem mem_bUnion {b : β} : b ∈ s.bUnion t ↔ ∃a∈s, b ∈ t a :=
 by simp only [mem_def, bUnion_val, mem_erase_dup, mem_bind, exists_prop]
 
+@[simp] lemma coe_bUnion : ↑(s.bUnion t) = (⋃ x ∈ (↑s : set α), ↑(t x) : set β) :=
```

Comment:
```suggestion
@[simp] lemma coe_bUnion : ↑(s.bUnion t) = (⋃ x ∈ s, ↑(t x) : set β) :=
```

Reply:
Note that I got this from the original formulation, so it might break some stuff.

Reply:
Yup, it's breaking stuff. Reverting.